poc/tls/gen-cert.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+openssl genrsa -out server.key 2048
```

Comment:
Can we remove the dependency on openssl?

Reply:
May be of use https://godoc.org/github.com/spacemonkeygo/openssl